x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/transforms/pivot/Aggregations.java

```diff
@@ -25,7 +25,11 @@ private Aggregations() {}
      */
     enum AggregationType {
         AVG("avg", "double"),
-        MAX("max", null);
+        CARDINALITY("cardinality", "long"),
```

Comment:
The aggregation names are redundant here. `getName()` could return `name().toLowerCase(Locale.ROOT)`.

Reply:
yeah, thought the same. But did not change it, because that's not part of this PR. 

I think I keep this for a later re-factoring, while this is redundant right now, we likely need more meta information for other purposes. One argument to keep this: if we want to have "median" we need to map to the "percentiles" aggregation.